main.go

```diff
@@ -0,0 +1,240 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/events"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/client"
+	"github.com/urfave/cli"
+)
+
+const (
+	imageListPutImagesPath = "/images"
+)
+
+var (
+	tagRegexp = regexp.MustCompile(`:([\w][\w.-]{0,127})$`)
+	logger    = log.New(os.Stderr, "", log.Ldate|log.Ltime|log.Lshortfile)
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "imagelist-docker-events"
+	app.Version = "v0.0.1"
+	app.Usage = "poll docker for image push events and add them to imagelist service"
```

Comment:
What is `them` in this context? The push events?

(This could do with a tad more clarifying, if possible).